javalib/src/main/scala/java/net/InetAddress.scala

```diff
@@ -563,11 +564,15 @@ private[net] trait InetAddressBase {
   }
 }
 
-object InetAddress extends InetAddressBase
+object InetAddress extends InetAddressBase {
+  // cached host values are discarded after this amount of time
+  private val HostTimeoutSeconds: Double = 30.0
+}
 
 class InetAddress private[net] (
     ipAddress: Array[Byte],
-    private var host: String
+    private var host: String,
+    private var hostLastAccessed: time_t = 0
```

Comment:
We do never need this field in the constructor, we can move it to be simply a field within the class.

Reply:
Oops, it initially was in the constructor and I forgot to move it. Thanks!